examples/pytorch/graphsaint/README.md

```diff
@@ -26,74 +26,120 @@ All datasets used are provided by Author's [code](https://github.com/GraphSAINT/
+| --- | --- | --- | --- | --- | --- |
+| Paper | 0.981±0.007 | 0.510±0.002 | 0.966±0.001 | 0.653±0.003 | 0.807±0.001 |
+| Running | 0.9810 | 0.5066 | 0.9656 | 0.6531 | 0.8071 |
+| DGL_offline | 0.9817      | 0.4456      | 0.9655 | 0.6530 | 0.8034 |
+| DGL_online | 0.9815 | 0.4508 | 0.9653 | 0.6516 | 0.7756 |
 
 #### Random walk sampler
-| Method | PPI | Flickr |
-| --- | --- | --- |
-| Paper | 0.981±0.004 | 0.511±0.001 |
-| Running | 0.9812 | 0.5104 |
-| DGL | 0.9818 | 0.5018 |
+| Method      | PPI         | Flickr      | Reddit      | Yelp        | Amazon      |
+| --- | --- | --- | --- | --- | --- |
+| Paper | 0.981±0.004 | 0.511±0.001 | 0.966±0.001 | 0.653±0.003 | 0.815±0.001 |
+| Running | 0.9812 | 0.5104 | 0.9648      | 0.6527      | 0.8131      |
+| DGL_offline | 0.9833      | 0.4592      | 0.9582      | 0.6514      | 0.8178   |
+| DGL_online | 0.9820 | 0.4606 | 0.9572      | 0.6508      | 0.8157   |
 
 ### Sampling time
 
+- Here sampling time includes consumed time of pre-sampling subgraphs and calculating normalization coefficients in the beginning.
+
 #### Random node sampler
 
-| Method | PPI | Flickr |
-| --- | --- | --- |
-| Sampling(Running) | 0.77 | 0.65 |
-| Sampling(DGL) | 0.24 | 0.57 |
-| Normalization(Running) | 0.69 | 2.84 |
-| Normalization(DGL) | 1.04 | 0.41 |
+| Method      | PPI  | Flickr | Reddit | Yelp | Amazon |
+| --- | --- | --- | --- | --- | --- |
+| Running | 1.46 | 3.49 | 19 | 59.01 | 978.62 |
+| DGL_offline | 2.51 | 1.12 | 27.32 | 60.15 | 929.24 |
+| DGL_online | 3.17 | 1.11 | 27.82 | 59.42 | 991.68 |
 
 #### Random edge sampler
 
-| Method | PPI | Flickr |
-| --- | --- | --- |
-| Sampling(Running) | 0.72 | 0.56 |
-| Sampling(DGL) | 0.50 | 0.72 |
-| Normalization(Running) | 0.68 | 2.62 |
-| Normalization(DGL) | 0.61 | 0.38 |
+| Method      | PPI  | Flickr | Reddit | Yelp | Amazon |
+| --- | --- | --- | --- | --- | --- |
+| Running | 1.4 | 3.18 | 13.88 | 39.02 | 164.35 |
+| DGL_offline | 3.08 | 1.91 | 53.59 | 48.94 | 1469.52 |
+| DGL_online | 3.04 | 1.87 | 52.01 | 48.38 | 1432.79 |
 
 #### Random walk sampler
 
-| Method | PPI | Flickr |
-| --- | --- | --- |
-| Sampling(Running) | 0.83 | 1.22 |
-| Sampling(DGL) | 0.28 | 0.63 |
-| Normalization(Running) | 0.87 | 2.60 |
-| Normalization(DGL) | 0.70 | 0.42 |
+| Method      | PPI  | Flickr | Reddit | Yelp | Amazon |
+| --- | --- | --- | --- | --- | --- |
+| Running | 1.7 | 3.82 | 16.97 | 43.25 | 355.68 |
+| DGL_offline | 3.10 | 2.18 | 10.57 | 21.52 | 153.93 |
+| DGL_online | 3.05 | 2.13 | 11.01 | 22.23 | 151.84 |
+
+## Test std of sampling time
+
+- We've run experiments 10 times repeatedly to test average and standard deviation of sampling time. Here we just test sampling time without model performance. Moreover, for efficient testing, the hardware and config employed here are not the same as the experiments above, so the sampling time might be a bit different from that above. But we keep the environment consistent in all experiments below.
```

Comment:
Is the definition of sampling time same as the section above?

Reply:
"without model performance" -> "without evaluating model performance" or "without training the model to the end"?

Reply:
How different is the config?

Reply:
> Is the definition of sampling time same as the section above?

The sampling time in the section above includes sampling and normalization time like what I specified in the section above. I'll modify the titles and annotations in this section so as to disambiguate.

Reply:
> How different is the config?

In this section I switch to another larger machine to test time. So I modified `num_workers_sampler` and `batch_size_norm` to make use of machine fully. But the number of subgraphs that used in training phase is not changed, so the performance of model should be the same. Other parameters are kept consistent across the section above and this section.

Reply:
Ok, then you might want to briefly explain the differences.